Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -601,56 +601,75 @@ function vtkRenderWindowInteractor(publicAPI, model) {
         inputSource.gripSpace == null
           ? null
           : xrFrame.getPose(inputSource.gripSpace, xrRefSpace);
-      const gp = inputSource.gamepad;
+
+      const targetRayPose =
+        inputSource.gripSpace == null
+          ? null
+          : xrFrame.getPose(inputSource.targetRaySpace, xrRefSpace);
```

Comment:
`inputSource.gripSpace ?? xrFrame.getPose(inputSource.targetRaySpace, xrRefSpace);`

Reply:
It does not work, for an obscure reason. Thus the lines above which also uses a ternary